pkg/chains/formats/slsa/v2/intotoite6v2_test.go

```diff
@@ -0,0 +1,381 @@
+/*
+Copyright 2021 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v2
+
+import (
+	"testing"
+	"time"
+
+	"github.com/tektoncd/chains/pkg/chains/formats"
+	"github.com/tektoncd/chains/pkg/chains/formats/slsa/v2/taskrun"
+	"github.com/tektoncd/chains/pkg/chains/objects"
+	"github.com/tektoncd/chains/pkg/config"
+	"github.com/tektoncd/chains/pkg/internal/objectloader"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/in-toto/in-toto-golang/in_toto"
+	slsa "github.com/in-toto/in-toto-golang/in_toto/slsa_provenance/v0.2"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/pod"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	logtesting "knative.dev/pkg/logging/testing"
+)
+
+var (
+	e1BuildStart    = time.Unix(1617011400, 0)
+	e1BuildFinished = time.Unix(1617011415, 0)
+)
+
+func TestTaskRunCreatePayload1(t *testing.T) {
```

Comment:
Looking at the example payloads side by side, I think we may be missing result output values?

Reply:
You mean task results right? If so, I believe those are not required fields in the provenance according to [TEP-0122](https://github.com/tektoncd/community/pull/820). That's because it is an output of the task/pipeline and so it's not something we need to capture in order to reproduce the build or anything. 
If we still need it in the provenance for other purposes, we can open an issue and I can implement it.

Reply:
Without the results, how do you know if the reproduced build resulted in the same output? 😅 

I'm okay if we want to implement this in pieces, but we should set expectations around readiness/completeness of the v2 type (see https://github.com/tektoncd/chains/pull/663#discussion_r1106248938 for related discussion).

Reply:
I was thinking that we would have that information in the subjects. ie. A reproducible build should lead to the same subjects but I'm happy to include results as well since I completely agree with the sentiment.

Reply:
Also, where should I store the taskrun results? I don't think invocation parameters/environment is the right place. I will place it in buildConfig for now unless there are concerns.

Reply:
I placed it in the buildConfig.